src/Service.GraphQLBuilder/Queries/QueryBuilder.cs

```diff
@@ -71,7 +72,7 @@ public static class QueryBuilder
                     }
                     else
                     {
-                        IEnumerable<string> rolesAllowedForRead = IAuthorizationResolver.GetRolesForOperation(entityName, operation: Operation.Read, entityPermissionsMap);
+                        IEnumerable<string> rolesAllowedForRead = IAuthorizationResolver.GetRolesForOperation(entityName, operation: EntityActionOperation.Read, entityPermissionsMap);
```

Comment:
below this, you could also update away from rolesAllowedForRead.Count() > 0 to the .Any() you've been doing :)